includes/subscriptions.php

```diff
@@ -234,7 +234,25 @@ function delete_subscriptions( $post_id ) {
  * @since  1.0
  */
 function send_notifications( $post_id ) {
-	if ( ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) || wp_is_post_revision( $post_id ) || ! current_user_can( 'edit_post', $post_id ) ) {
+	if ( ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE ) || wp_is_post_revision( $post_id ) ) {
+		return;
+	}
+
+	if ( ! wp_doing_cron() ) { // @codingStandardsIgnoreLine `wp_doing_cron(..)` is a WP function
+		/**
+		 * Add possibility to send notification in background
+		 *
+		 * @param bool  false   Whether run 'send notification' in background or not, default 'false'
+		 * @param array $params request data
+		 */
+		$send_notification_in_background = apply_filters( 'dt_send_notification_allow_in_background', false, $post_id );
+
+		if ( true === $send_notification_in_background ) {
+			return;
+		}
+	}
+
+	if ( ! current_user_can( 'edit_post', $post_id ) ) {
```

Comment:
can this stay as part of the logic check on L:237?

Reply:
We don't need this check when running in background as in cron there is no current in user.